dash/development/_r_components_generation.py

```diff
@@ -163,7 +142,7 @@ def generate_class_string(name, props, project_shortname, prefix):
         '\'{}\''.format(p)
         for p in prop_keys
         if '*' not in p and
-        p not in ['setProps', 'dashEvents', 'fireEvent']
+        p != 'setProps' + ['**kwargs']
```

Comment:
That doesn't work.
```python
>>> 'setProps' + ['**kwargs']
Traceback (most recent call last):
  File "<input>", line 1, in <module>
TypeError: must be str, not list
```

```suggestion
        p not in ['setProps', '**kwargs']
```

Reply:
Oh that's not quite it either... the `p not in [...]` or `p != ...` clause is a filter in an iterator - in the python generator code: 
https://github.com/plotly/dash/blob/e72f483c85aaa91e4316a548ed83ab40a6819452/dash/development/_py_components_generation.py#L118-L126

note the extra `[...]` from immediately after `.join(` to right before `+ ['**kwargs']` that turns this into a list that we can then append `'**kwargs'` to by adding a second list. I'm assuming in R `**kwargs` doesn't mean anything, but there's some other feature to play the role of allowing arbitrary extra named arguments?

Reply:
> I'm assuming in R **kwargs doesn't mean anything, but there's some other feature to play the role of allowing arbitrary extra named arguments?

@alexcjohnson Correct. For the wildcards, I've used ellipsis (aka `\dots`, `...`) for this purpose. It seems to be working to pass in an arbitrary number (including zero) of wildcard arguments.